admin/sql/updates/2021-04-12-new-music-services-table.sql

```diff
@@ -0,0 +1,25 @@
+BEGIN;
```

Comment:
maybe add a suffix to the dates to ensure that we know to apply this one before the migrate one